# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/Localization.webidl
# Commit: 3404fd6725f1
# Full Hash: 3404fd6725f15d1253a7f76e3937185bd353f263
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-02 21:43:56
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r c9cfbba15d90 -r 3404fd6725f1 dom/webidl/Localization.webidl
--- a/dom/webidl/Localization.webidl	Mon Aug 02 09:54:07 2021 +0000
+++ b/dom/webidl/Localization.webidl	Mon Aug 02 09:54:07 2021 +0000
@@ -92,16 +92,13 @@
    *                             default generators provided by Gecko.
    */
   [Throws]
-  constructor(sequence<DOMString> aResourceIds,
-              optional boolean aSync = false,
-              optional BundleGenerator aBundleGenerator = {});
+  constructor(sequence<UTF8String> aResourceIds,
+              optional boolean aSync = false);
 
   /**
    * A method for adding resources to the localization context.
-   *
-   * Returns a new count of resources used by the context.
    */
-  unsigned long addResourceIds(sequence<DOMString> aResourceIds);
+  void addResourceIds(sequence<DOMString> aResourceIds);
 
   /**
    * A method for removing resources from the localization context.